src/harness/parallel/worker.ts

```diff
@@ -12,6 +12,11 @@ namespace Harness.Parallel.Worker {
             testList.length = 0;
         }
         reportedUnitTests = true;
+        if (testList.length) {

```

Comment:
what's this change for? were we not running some unit tests before?

Reply:
No, this is just to make it so unit test execution duration isn't included in the duration for the first normal test.